smartsim/_core/control/controller.py

```diff
@@ -312,7 +335,9 @@ def init_launcher(self, launcher: str) -> None:
         else:
             raise TypeError("Must provide a 'launcher' argument")
 
-    def _launch(self, manifest: Manifest) -> None:
+    def _launch(
+        self, exp_name: str, exp_path: str, manifest: Manifest
+    ) -> LaunchedManifest[t.Tuple[str, Step]]:
```

Comment:
Missed this on first review, can you please update the docstring to reflect the new function signature?

Reply:
Added `exp_path` & `exp_name` params in docstring